microshift_getting-started/microshift-architecture.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="microshift-architecture"]
+= Architecture
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-architecture
+
+toc::[]
+
```

Comment:
It seems like there should be an abstract here between the "Architecture" heading and the "benefits" heading.

Reply:
Reworked it a bit. LMK what you think.